Description: Document the exit status of quilt
 This includes the behavior induced by the return2 patch, that is
 still not included upstream.
Author: Martin Quinson <mquinson@debian.org>
Last-Update: 2012-12-27
Bug-Debian: http://bugs.debian.org/587001
Upstream-status: not reported, waits for the integration of return2

Index: quilt.git/doc/quilt.1.in
===================================================================
--- quilt.git.orig/doc/quilt.1.in	2012-12-27 14:48:23.390034278 +0100
+++ quilt.git/doc/quilt.1.in	2012-12-27 14:57:49.217355247 +0100
@@ -122,6 +122,13 @@
 
 Print the version number and exit immediately.
 
+.SH EXIT STATUS
+
+The exit status is 0 if the subcommand was successfully executed, and
+1 in case of error. An exit status of 2 denotes that quilt did not do
+anything to complete the command; This happens for example when asking
+to push when the whole stack is already pushed. This behavior is
+intended to ease the scripting around quilt.
 
 .SH EXAMPLE OF WORKING TREE
 
@@ -141,9 +148,7 @@
 .RE
 .fam T
 
-.SH EXAMPLE
-
-Please refer to the pdf documentation for an example.
+Please refer to the pdf documentation for a full example of use.
 
 .SH CONFIGURATION FILE
 
